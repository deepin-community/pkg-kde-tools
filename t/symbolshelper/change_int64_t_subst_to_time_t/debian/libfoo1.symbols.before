# SymbolsHelper-Confirmed: 1.0 amd64 armhf
libfoo.so.1 libfoo1 #MINVER#
 (subst)_Z3foo{int64_t} 1.0
