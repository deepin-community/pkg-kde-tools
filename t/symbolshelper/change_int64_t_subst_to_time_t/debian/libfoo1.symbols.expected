# SymbolsHelper-Confirmed: 1.0 amd64 armhf i386
libfoo.so.1 libfoo1 #MINVER#
 (subst)_Z3foo{time_t} 1.0
